torch/quantization/fx/quantize.py

```diff
@@ -1096,10 +1112,10 @@ def _find_quants(self, graph: Graph, matches: Dict[str, MatchResult],
           - matches: output of self._find_matches function
 
         Outputs a map of
-         node_name -> (QuantizeHandler instance (always DefaultQuantizeHandler),
+         node_name -> list of (QuantizeHandler instance (always DefaultQuantizeHandler),
```

Comment:
can we make sure that the docblock explains why this is a list

Reply:
sure